### ISSUE TITLE
SN - le symbole de la monnaie dans les notifications n'est pas le bon

### ISSUE THREAD

**Issue by ioknox** [fablab-chaux-de-fonds/fab-manager#3]

**Comment by ioknox** [comment]
@matbgn Tu n'avais pas eut un problème similaire pour AT3FLO ?

**Comment by matbgn** [comment]
Yep voir l'explication que j'avais donné sur le Slack de la Chaux-de-Fonds.
Mais comme mon compte a été désactivé je ne sais pour quelle raison je peux
pas vraiment te dire où exactement.

**Comment by ioknox** [comment]
J'ai posé la question pour ton compte...

Actuellement j'ai ça :
RAILS_LOCALE=fr-CH
APP_LOCALE=fr
MOMENT_LOCALE=fr
SUMMERNOTE_LOCALE=fr-FR
ANGULAR_LOCALE=fr-ch
MESSAGEFORMAT_LOCALE=fr
FULLCALENDAR_LOCALE=fr

**Comment by matbgn** [comment]
Qu'obtiens-tu quand tu intéroges les paramètres de Stripe :

rails c
Rails.application.secrets.stripe_currency
exit

**Comment by ioknox** [comment]
irb(main):001:0> Rails.application.secrets.stripe_currency => "chf"

**Comment by matbgn** [comment]
Ok est-ce que tu peux lancer les commandes suivantes et reporter ici les résultats :

1. Lancer la console rails
```
docker exec -it fabmanager_fabmanager_1 bash
bundle
rails c
```
2. Unité monétaire (rails)
```
I18n.t 'number.currency.format.unit'
```
3. Conversion d'un montant en devises
```
include ActionView::Helpers::NumberHelper
number_to_currency(12)
```

Est-ce que tu as fait une nouvelle facture depuis ? En effet, le changement de locale ne déclenche pas la régénération de toutes les factures d'un point de vue compta c'est la règle d'ailleurs, donc celles qui ont été générés précédemment restent tels quels.

Par contre tu peux forcer la régénération manuellement si tu le souhaites. Pour ça :

```
docker exec -it fabmanager_fabmanager_1 bash
bundle
rake fablab:regenerate_invoices[year,month]
# par exemple rake fablab:regenerate_invoices[2017,1] va régénéré toutes les factures de janvier 2017 (et seulement celles-là).
```

> Avec le support de sylvainbx de La Casemate à l'époque.

**Comment by ioknox** [comment]
Merci pour ces réponses. *Seule les notifications sont en euro*.

2. => "sFr."
3. => "12.00 sFr."
